Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst

```diff
@@ -0,0 +1,129 @@
+.. An error will alert you of identical labels during the build process.
+
+#####################
+AuxTel LOTO Procedure
+#####################
+
+.. _AuxTel-LOTO-procedure-Overview:
+
+Overview
+^^^^^^^^
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+AuxTel must be under LOTO (lock-out tag-out) any time physical work is undertaken on the mount or the dome. 
+Working with environmental sensors, on the truss, mirrors, or anywhere in the dome where unexpected movement could cause a safety hazard, the telescope must be locked out. 
+Trained summit support staff may use this procedure if they meet all of the preconiditon requirements. 
+
+.. _AuxTel-LOTO-procedure-Precondition:
+
+Precondition
+^^^^^^^^^^^^
+
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+Before putting the AuxTel under LOTO, you must have completed the :ref:`LOTO safety training <Daytime-Operations-Safety-Control-Safety-Systems-LOTO>` and familiarized yourself with the :ref:`Control Safety Systems <Daytime-Operations-Safety-Control-Safety-Systems>` document. 
+You should have your own unique lock and key with your information identified, and this work must be coordinated with the day crew. 
+
+- The telescope must be parked in a safe position, and all CSCs in STANDBY. 
+
+    - If the ATDome is needed while the AuxTel is under LOTO, this must be verified with the safety team and day crew.
+
+- The work should be scheduled in the `SUMMIT JIRA calendar <https://jira.lsstcorp.org/secure/DoItBetterCalendar.jspa>`__ and day crew as well as nighttime support should be aware of the plan. 
+
+.. _AuxTel-LOTO-procedure-Post-Condition:
+
+Post-Condition
+^^^^^^^^^^^^^^
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+The drives are power off, and the AuxTel Control cabinet is locked from the outside. 
```

Comment:
Typo: "powered", not sure if the comma is needed here either.

Reply:
👍